bitnami/opencart/Chart.yaml

```diff
@@ -29,4 +29,4 @@ name: opencart
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/opencart
   - https://opencart.com/
-version: 12.2.11
+version: 12.2.12
```

Comment:
Please bump a minor version for new features (`12.3.0`)

Reply:
Thank for your review. I had already bumped a minor version.